app/views/subscriptions/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
should we name our forms in a more explicit way, i.e. _add_sub_form?

Reply:
yes will update